extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -436,6 +437,9 @@ export class SchemaCompareOptionsDialog {
 	protected async execute() {
 		this.SetDeploymentOptions();
 		this.SetObjectTypeOptions();
+		if (this.optionsChanged) {
+			vscode.window.showInformationMessage(localize('schemaCompareOptions.recompareMessage', 'Options have changed. Press Compare to see the comparison.'));
+		}
```

Comment:
I think we should set this.optionsChanged if reset button was clicked too - what do you think? (SSDT does that - if reset is click with or without options changed - the message comes)

Reply:
Yes, this.optionsChanged gets set to true in reset()